bin/env

```diff
@@ -28,7 +28,12 @@ while ( @ARGV && $ARGV[0] =~ /^-/ ) {
 	if ( $arg eq '-i' ) {
 		%ENV = ();
 	} elsif ( $arg =~ /^-u(.*)/ ) {
-		delete $ENV{ length($1) ? $1 : shift };
+		my $val = length $1 ? $1 : shift;
```

Comment:
A bit ugly, but until we switch to Get::Opts, this is fine. I've done it like this before but I've never been happy with it.